openstates/me/bills.py

```diff
@@ -194,21 +194,31 @@ def scrape_bill(self, bill):
         if srow:
             bill['subjects'] = [s.strip() for s in srow if s.strip()]
 
+        # Attempt to find link to bill text/documents.
         ver_link = page.xpath("//a[contains(@href, 'display_ps.asp')]")[0]
         ver_url = ver_link.get('href')
+
         try:
             ver_html = self.get(ver_url, retry_on_404=True).text
```

Comment:
@loandy, not necessary to change here, but check out the `lxmlize()` function, a utility that's really really useful in our scraping code:
https://github.com/sunlightlabs/openstates/blob/master/openstates/utils/lxmlize.py